lib/Runtime/Library/RuntimeLibraryPch.h

```diff
@@ -95,8 +95,16 @@
 #include "Language/CacheOperators.inl"
 
 #ifdef INTL_ICU
+
+#ifndef U_STATIC_IMPLEMENTATION
 #define U_STATIC_IMPLEMENTATION
+#endif
+
+#ifdef U_SHOW_CPLUSPLUS_API
+#undef U_SHOW_CPLUSPLUS_API
 #define U_SHOW_CPLUSPLUS_API 0
+#endif
```

Comment:
These defines are only necessary if this compilation unit uses the ICU headers. I think you can safely remove this block now.

Reply:
Yeah I think they were necessitated by moving the includes to intl.h but that doesn't work on Linux so I am reverting it now.